ui/src/Context/NotificationContext.js

```diff
@@ -0,0 +1,34 @@
+import { createContext, useContext, useState } from "react";
+import { NOTIFICATION_TIMEOUT } from "../Constant/constant";
+
+export const NotificationContext = createContext({
+  createNotifcation() {},
+  notificationQue: [],
+});
+
+export const useNotificationContext = () => useContext(NotificationContext);
+
+const NotificationProvider = ({ children }) => {
+  const [notificationQue, setNotificationQue] = useState([]);
+
+  const createNotifcation = (notificationData) => {
```

Comment:
if this is the `notification content`, can we rename it to `content` or `data` only? As we have only notification data and not any other data here. wyt?